README.md

```diff
@@ -46,20 +46,42 @@ POSIX threads library needs LSE atomic instructions.  LSE is important for
 locking and thread synchronization routines.  The dynamic linker on Linux can
 detect CPU capabilities and load libc built with LSE.  For example, Ubuntu 20.04
 contains the libc6-lse package.  When installed on a Graviton2 system, all
-applications will link against this library.  ldd command line utility displays
-the path of the linked dynamic libraries.  If libpthreads.so is in atomics
-directory, the library uses LSE atomic instructions.  Amazon Linux 2 will soon
-provide a similar mechanism to select an LSE version of libc.
+applications will link against this library.  __Action item: check that an
```

Comment:
instead of this maybe we should just mention the OSes with the optimization and add as we go.

Reply:
I added a sentence to enumerate the systems we know of that distribute a libc compiled with LSE.